src/printer/index.js

```diff
@@ -777,7 +819,7 @@ function genericPrint(path, options, print) {
 
     default:
       // eslint-disable-next-line no-console
-      console.error("Unknown Ruby Node:", n);
+      console.error("Unhandled node within src/printer/index.js: ", n);
```

Comment:
I decided to make it clear which visitor was having issues parsing trees; I'd prefer not to hard code the path here like I have done with the updated Ruby error, but I imagine this won't change for a very long time - or it will be immediately obvious either way.

Happy to make variations here as appropriate.